exporter/zipkin/test/test_helper.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require 'simplecov'
+SimpleCov.start
+
+require 'opentelemetry/exporter/zipkin'
+require 'minitest/autorun'
+require 'webmock/minitest'
+
+def with_env(new_env)
+  env_to_reset = ENV.select { |k, _| new_env.key?(k) }
+  keys_to_delete = new_env.keys - ENV.keys
+  new_env.each_pair { |k, v| ENV[k] = v }
+  yield
+ensure
+  env_to_reset.each_pair { |k, v| ENV[k] = v }
+  keys_to_delete.each { |k| ENV.delete(k) }
+end
```

Comment:
This is the fifth copy of `with_env`. It might be worth extracting it to `opentelemetry-common`.

Reply:
Makes sense to me but not sure to where would we extract this, as it's only for tests.  `common/tests/test_helper.rb` maybe? or would it belong in something like `common/lib/opentelemetry/test_utils/ `? wdyt?

Reply:
added a small helpers file to `common` but i didn't include it in the `Common` module as I figure test helpers shouldn't be available for use from `opentelemetry/common`...maybe an anti patten, certainly if there's a more 'ruby' way to accomplish this i'm happy to update

Reply:
For now, I think we decided to keep both common test helpers and common ... non-test helpers in `opentelemetry/common`. We can discuss splitting them out later, but so far there hasn't been enough of either to warrant splitting the gem.